index.html

```diff
@@ -42,5 +42,42 @@ <h1 class="title">New strategies that lead to a growing in sales</h1>
         </button>
       </div>
     </section>
+    -->
+
+    <!--====benefits-section====-->
+
+    <section class="benefits-section">
+      <div class="benefits-section-title">
+        <h3 class="benefits-section-subtitle">Benefits of coaching</h3>
+      </div>
+      <div class="benefits-section-clander">
+        <div class="benefits-section-one">
+          <p class="benefits-section-num-one">01</p>
+          <div class="benefits-section-subtitle-one">
+            <img src="images/benefist-section/Vector 19.svg" alt="line" />
+            <h2 class="benefits-section-header-one">Schedules</h2>
+            <p class="benefits-section-subone">Ut mattis vulputate id enim lorem eu nullam. Sceleris lobortis phasellus.</p>
+          </div>
+        </div>
+        <div class="benefits-section-two">
+          <p class="benefits-section-num-two">02</p>
+          <div class="benefits-section-subtitle-two">
+            <img src="images/benefist-section/Vector 19.svg" alt="line" />
+            <h2 class="benefits-section-header-two">Organisation</h2>
+            <p class="benefits-section-subtwo">Ut mattis vulputate id enim lorem eu nullam. Sceleris lobortis phasellus.</p>
+          </div>
+        </div>
+        <div class="benefits-section-three">
+          <p class="benefits-section-num-three">03</p>
+          <div class="benefits-section-subtitle-three">
+            <img src="images/benefist-section/Vector 19.svg" alt="kline" />
+            <h2 class="benefits-section-header-three">Time & Money</h2>
+            <p class="benefits-section-subthree">Ut mattis vulputate id enim lorem eu nullam. Sceleris lobortis phasellus.</p>
+          </div>
+        </div>
```

Comment:
One two three olarak ayrı ayrı belirttiğin class isimleri için aynı özelliklere sahip olan elementlere aynı class adını vermelisin, örn `.card-num ... .card-title ... .card-description` gibi. Çünkü üç karttaki sayıların, başlıkların ve altındaki açıklamaların özellikleri birbiriyle aynı.